kong/plugins/cors/handler.lua

```diff
@@ -107,22 +107,19 @@ function CorsHandler:access(conf)
   CorsHandler.super.access(self)
 
   if req_get_method() == "OPTIONS" then
+    -- Don't add any response header because we are delegating the preflight to
+    -- the upstream API (conf.preflight_continue=true)
```

Comment:
nitpick- this comment isn't quite accurate anymore. could be something like:

```
-- Don't add any response header because we are delegating the preflight to
-- the upstream API (conf.preflight_continue=true), or because we already added them all